api/order.go

```diff
@@ -162,17 +186,45 @@ func (a *API) setUserIDFromToken(tx *gorm.DB, user *models.User, order *models.O
 	return nil
 }
 
-func (a *API) createLineItems(tx *gorm.DB, order *models.Order, items []models.LineItem) *HTTPError {
+func (a *API) createLineItems(ctx context.Context, tx *gorm.DB, order *models.Order, items []*OrderLineItem) *HTTPError {
+	sem := make(chan int, MaxConcurrentLookups)
```

Comment:
I use something like that in the logbase - but for testing that I also spin up an instance in the ci script. 

https://github.com/netlify/logbase/blob/master/storage/store.go